rust/lanzatool/src/signature.rs

```diff
@@ -31,7 +32,7 @@ impl KeyPair {
         let output = Command::new("sbsign").args(&args).output()?;
 
         if !output.status.success() {
-            print!("{:?}", output.stderr);
+            std::io::stderr().write_all(&output.stderr).unwrap();
```

Comment:
Why not propagate the error (with some context)?

Reply:
See #43